samples/MobileBuyIntegration/app/src/main/java/com/shopify/checkout_sdk_mobile_buy_integration_sample/common/logs/Logger.kt

```diff
@@ -87,8 +88,7 @@ class Logger(private val logDb: LogDatabase) {
         )
     }
 
-    @OptIn(DelicateCoroutinesApi::class)
-    private fun insert(logLine: LogLine) = GlobalScope.launch(Dispatchers.IO) {
```

Comment:
Overall we should try to avoid `GlobalScope` when possible (that's why its usage requires `DelicateCoroutinesApi` annotation). By passing the scope as constructor param, we can even facilitate testing in the future (by providing a scope with immediate dispatcher)